.cspell.json

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "0.2",
+  "language": "en",
+  // words - list of words to be always considered correct
```

Comment:
⚠️ Comments in JSON are technically invalid and will alert in other linters. I think I would rather we just document what we do here at https://contributing.bitwarden.com/contributing/template-repository.